addon/mixins/route.js

```diff
@@ -412,7 +252,7 @@ const RouteMixin = Mixin.create({
       return;
     }
 
-    Ember.run.scheduleOnce('afterRender', this, 'infinityModelUpdated', {
+    run.scheduleOnce('afterRender', this, 'infinityModelUpdated', {
       lastPageLoaded: this.get('currentPage'),
       totalPages: this.get('_totalPages'),
```

Comment:
we should ensure we've got a deprecate for any developers who were using this notification